experimental/FTheoryTools/src/HypersurfaceModels/constructors.jl

```diff
@@ -23,12 +23,13 @@ function hypersurface_model(base::NormalToricVariety; completeness_check::Bool =
   set_coordinate_names(fiber_ambient_space, ["x", "y", "z"])
   D1 = 2 * anticanonical_divisor_class(base)
   D2 = 3 * anticanonical_divisor_class(base)
-  return hypersurface_model(base, fiber_ambient_space, D1, D2; completeness_check = completeness_check)
+  D3 = trivial_divisor_class(base)
+  return hypersurface_model(base, fiber_ambient_space, [D1, D2, D3]; completeness_check = completeness_check)
 end
 
 
 @doc raw"""
-    hypersurface_model(base::NormalToricVariety, fiber_ambient_space::NormalToricVariety, D1::ToricDivisorClass, D2::ToricDivisorClass; completeness_check::Bool = true)
+    hypersurface_model(base::NormalToricVariety, fiber_ambient_space::NormalToricVariety, fiber_twist_divisor_classes::Vector{ToricDivisorClass}; completeness_check::Bool = true)
 
 Construct a hypersurface model, for which the user can specify a fiber ambient space
 as well as divisor classes of the toric base space, in which the first two homogeneous
```

Comment:
This is no longer "the first two", right?

Reply:
I will chase that in the documentation in a separate PR.